model/linux/registry.yaml

```diff
@@ -16,3 +16,20 @@ groups:
         stability: experimental
         brief: "The Linux Slab memory state"
         examples: ["reclaimable", "unreclaimable"]
+  # linux.process.* attribute group
+  - id: registry.linux.process
+    type: attribute_group
+    brief: "Describes Linux Process attributes"
+    attributes:
+      - id: linux.process.cgroup
```

Comment:
You probably want to reference this attribute in process resource definition - https://github.com/open-telemetry/semantic-conventions/blob/main/model/process/resources.yaml

it seems to be a resource attribute in the collector receiver - https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml#L49

Reply:
Good point, discussed in System Semantic conventions SIG (21/11/2024) and we agreed on adding it into the resource definition.